mmv1/products/networkconnectivity/Group.yaml

```diff
@@ -0,0 +1,112 @@
+async:
+  type: 'OpAsync'
+  operation:
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result:
+    path: 'targetLink'
+  error:
+    path: 'error/errors'
+    message: 'message'
+examples:
+  - name: 'network_connectivity_group_basic'
+    primary_resource_id: 'primary'
+    vars:
+      hub_name: "hub1"
+      auto_accept_project_1_name: "foo"
+      auto_accept_project_2_name: "bar"
+description: The NetworkConnectivity Group resource
+parameters:
+  - name: 'hub'
+    type: ResourceRef
+    resource: 'Hub'
+    imports: 'id'
+    description: |
+      The name of the hub. Hub names must be unique. They use the following form: projects/{projectNumber}/locations/global/hubs/{hubId}
+    immutable: true
+    required: true
+    url_param_only: true
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+properties:
+  - name: name
+    type: Enum
+    description: 'The name of the group. Group names must be unique. They use the following form: `projects/{projectNumber}/locations/global/hubs/{hub}/groups/{groupId}`'
+    diff_suppress_func: 'tpgresource.CompareSelfLinkOrResourceName'
+    required: true
+    immutable: true
+    custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.tmpl'
+    enum_values:
+      - 'default'
+      - 'center'
+      - 'edge'
+  - name: createTime
+    type: String
+    description: Output only. The time the hub was created.
+    output: true
+  - name: updateTime
+    type: String
+    description: Output only. The time the hub was last updated.
+    output: true
+  - name: labels
+    type: KeyValueLabels
+    description: Optional labels in key:value format. For more information about labels, see [Requirements for labels](https://cloud.google.com/resource-manager/docs/creating-managing-labels#requirements).
+  - name: description
+    type: String
+    description: An optional description of the hub.
+  - name: uid
+    type: String
+    description: Output only. The Google-generated UUID for the group. This value is unique across all group resources. If a group is deleted and another with the same name is created, the new route table is assigned a different uniqueId.
+    output: true
+  - name: state
+    type: String
+    description: Output only. The current lifecycle state of this hub.
+    output: true
+    exactly_one_of:
+      - STATE_UNSPECIFIED
+      - CREATING
+      - ACTIVE
+      - DELETING
+    default_value: null
+  - name: autoAccept
+    type: NestedObject
+    description: Optional. The auto-accept setting for this group.
+    immutable: true
+    properties:
+      - name: autoAcceptProjects
+        type: Array
+        description: 'A list of project ids or project numbers for which you want to enable auto-accept. The auto-accept setting is applied to spokes being created or updated in these projects.'
+        min_version: ga
```

Comment:
Just want to double check that this field is only available in beta provider?

Reply:
so it is actually available in alpha, beta and ga; I have removed the min_version